backend/python/app/static/openapi.json

```diff
@@ -123,6 +149,20 @@
                   }
                 }
               }
+            },
+            "multipart/form-data": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "body": {
+                    "type": "string"
+                  },
+                  "file": {
+                    "type": "string",
+                    "format": "binary"
+                  }
```

Comment:
Reference for how to document a file field: https://swagger.io/docs/specification/describing-request-body/file-upload/